influxdb/src/main/scala/akka/stream/alpakka/influxdb/impl/InfluxDBFlowStage.scala

```diff
@@ -58,28 +61,70 @@ private[influxdb] class InfluxDBFlowStage[T, C](
   final class InfluxDBRecordLogic extends InfluxDBLogic {
 
     override protected def write(messages: immutable.Seq[InfluxDBWriteMessage[T, C]]): Unit =
-      messages.foreach {
-        case InfluxDBWriteMessage(point: Point, _) => {
-          influxDB.write(point)
+      messages
+        .filter {
+          case InfluxDBWriteMessage(point: Point, _, _, _) => {
```

Comment:
`point` and `others` actually aren't needed in these two `case`s, since we aren't using them in the subsequent blocks. We can just replace those with `_`.